themes/upbound/assets/scss/_color-modes.scss

```diff
@@ -1,45 +1,53 @@
 :root {
-    color-scheme: light dark;
-  }
+  color-scheme: light dark;
 
+  body{
+    background: var(--body-background);
+    color: var(--body-font-color);
+  }
 
-  :root[color-theme="light"] {
```

Comment:
Note: This file was badly formatted and I fixed the indentation which is now causing all the noise in this diff.